source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -528,6 +528,9 @@ def findOverlayClasses(self,clsList):
 		elif windowClassName.startswith("Chrome_"):
 			from . import chromium
 			chromium.findExtraOverlayClasses(self, clsList)
+		elif windowClassName == "ConsoleWindowClass":

```

Comment:
Rather than placing the code directly here, you can just place an entry in the _staticMap directory in this file. But as well as the window class, also  check for an MSAA role of ROLE_SYSTEM_CLIENT. This will ensure that the subclass is only used for the actual client of the window, and not the system menu or scrollbars etc.
 Something like:
```
("ConsoleWindowClass",oleacc.ROLE_SYSTEM_CLIENT):"winConsoleLegacy.WinConsoleLegacy",
```